### PR TITLE
Fix stage duration on stage overview

### PR DESCRIPTION
**Pull request by maheshp**
Issue: #8681 

Description: Fix stage duration on stage overview. It is same as the Stage details page.

Introduced v3 of Stage Instance API.

### CODE REVIEW

**Review by kritika-singh3 — CHANGES_REQUESTED** [review]
LGTM.
Should we add the new attributes to the history API as well??

File: api/api-stage-instance-v3/src/main/java/com/thoughtworks/go/apiv3/stageinstance/representers/StageRepresenter.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import com.thoughtworks.go.domain.StageResult;
+
+public class StageRepresenter {
+
+    public static void toJSON(OutputWriter jsonWriter, Stage stage) {
+        jsonWriter.add("name", stage.getName());
+        jsonWriter.add("counter", stage.getCounter());
+        jsonWriter.add("approval_type", stage.getApprovalType());
+        jsonWriter.add("approved_by", stage.getApprovedBy());
+        jsonWriter.add("created_time", stage.getCreatedTime().getTime());
```
[the remark this answers is not in the document]
Comment (maheshp) [reply api/api-stage-instance-v3/src/main/java/com/thoughtworks/go/apiv3/stageinstance/representers/StageRepresenter.java]: Changed it to `scheduled_at`.

**Comment by maheshp** [comment]
> Should we add the new attributes to the history API as well??

I did look at the history API, since the Stage JSON had minimal attributes compared to the Stage instance API I thought we can add it later if required.

**Review by kritika-singh3 — APPROVED** [review]

**Comment by kritika-singh3** [comment]
Verified on `GoCD Version: 20.9.0 (12318-b67833a26e7273f83019facdf22dfcfd290cdf4a)`

**Pull request merged**